src/me/StevenLawson/TotalFreedomMod/TFM_Util.java

```diff
@@ -144,7 +144,7 @@ public static void playerMsg(CommandSender sender, String message)
 
     public static void adminAction(String adminName, String action, boolean isRed)
     {
-        TFM_Util.bcastMsg(adminName + " - " + action, (isRed ? ChatColor.RED : ChatColor.AQUA));
+        TFM_Util.bcastMsg(adminName + " - " + action, (isRed ? ChatColor.YELLOW : ChatColor.GOLD));
```

Comment:
This should not really be changed. Currently there is the "isRed" and your then actually asking if it is yellow... Not really ideal and will cause confusion.